lib/model.py

```diff
@@ -1,17 +1,18 @@
 import autokeras as ak
 import tensorflow as tf
-from lib.base import IModel
 
+TABULAR = 'tabular'
 REGRESSION = 'regression'
 CLASSIFICATION = 'classification'
+FORECASTING = 'forecasting'
 
 SAVED_MODEL = 'saved_model'
 TFTRT = 'tftrt'
 TFLITE = 'tflite'
 ONNX = 'onnx'
 
 
-class StructuredDataModel(IModel):
```

Comment:
Looks like lib.base was merged in the past week. Do you remove it by mistake?

Reply:
Yes, reverted